src/org/omegat/gui/editor/EditorPopups.java

```diff
@@ -76,6 +80,14 @@ public static void init(EditorController ec) {
 
     private EditorPopups() {
     }
+    
+    /** 
+     * Translate menu item but ensuring it does not contain mnemonics,
+     * because they do not work with context menus
+     **/
+    public static String translateMenu(String id) {
```

Comment:
Could we name this method something else?
It seems to me like if we can avoid naming things "translate", we should try to do that, considering lots of things in OmegaT already "translate" in one way or another.

Reply:
Could you consider it is as private method?
I think we can call `Mnemonics.removeMnemonics` from `DictionaryPopup` class and `MultipleTranslationPopup` class.

Reply:
The goal of the method was to ensure that all popups which come to the same editor receive the same treatments, in case we later need to add something more than only removing the mnemonics.
But considering the polemics it created, I prefer to remove it completely.
Corrected now.